diff --git a/node_modules/@storybook/addon-storyshots/dist/frameworks/rn/loader.js b/node_modules/@storybook/addon-storyshots/dist/frameworks/rn/loader.js
index 3b098ce..6f6f21b 100644
--- a/node_modules/@storybook/addon-storyshots/dist/frameworks/rn/loader.js
+++ b/node_modules/@storybook/addon-storyshots/dist/frameworks/rn/loader.js
@@ -29,11 +29,11 @@ function configure(options, storybook) {
 
   const resolvedConfigPath = _path.default.resolve(configPath);
 
-  require.requireActual(resolvedConfigPath);
+  jest.requireActual(resolvedConfigPath);
 }
 
 function load(options) {
-  const storybook = require.requireActual('@storybook/react-native');
+  const storybook = jest.requireActual('@storybook/react-native');
 
   configure(options, storybook);
   return {
