diff --git a/node_modules/react-native/Libraries/LogBox/Data/LogBoxData.js b/node_modules/react-native/Libraries/LogBox/Data/LogBoxData.js
index 1076d26..12364ea 100644
--- a/node_modules/react-native/Libraries/LogBox/Data/LogBoxData.js
+++ b/node_modules/react-native/Libraries/LogBox/Data/LogBoxData.js
@@ -23,6 +23,10 @@ import type {
 import parseErrorStack from '../../Core/Devtools/parseErrorStack';
 import type {ExtendedError} from '../../Core/Devtools/parseErrorStack';
 import NativeLogBox from '../../NativeModules/specs/NativeLogBox';
+
+// https://github.com/facebook/react-native/pull/30027
+const ExceptionsManager = require('../../Core/ExceptionsManager');
+
 export type LogBoxLogs = Set<LogBoxLog>;
 export type LogData = $ReadOnly<{|
   level: LogLevel,
@@ -98,8 +102,6 @@ export function reportLogBoxError(
   error: ExtendedError,
   componentStack?: string,
 ): void {
-  const ExceptionsManager = require('../../Core/ExceptionsManager');
-
   error.forceRedbox = true;
   error.message = `${LOGBOX_ERROR_MESSAGE}\n\n${error.message}`;
   if (componentStack != null) {
diff --git a/node_modules/react-native/jest/setup.js b/node_modules/react-native/jest/setup.js
index 5f85d86..9375086 100644
--- a/node_modules/react-native/jest/setup.js
+++ b/node_modules/react-native/jest/setup.js
@@ -41,6 +41,14 @@ jest
       reportException: jest.fn(),
     },
   }))
+  // https://github.com/facebook/react-native/pull/30027
+  .mock('../Libraries/Core/ExceptionsManager', () => {
+    return {
+      handleException: (...args) => {
+        return;
+      },
+    };
+  })
   .mock('../Libraries/ReactNative/UIManager', () => ({
     AndroidViewPager: {
       Commands: {
